### ISSUE TITLE
I added quora and medium to social using the following code in _includes/social

### ISSUE THREAD

**Issue by mkakodkar** [alshedivat/al-folio#36]
```
{% if site.medium_username %}<a href="https://medium.com/@{{ site.medium_username }}" target="_blank" title="Medium"><i class="fab fa-medium"></i></a>{% endif %}
    {% if site.quora_username %}<a href="https://www.quora.com/profile/{{ site.quora_username }}" target="_blank" title="Quora"><i class="fab fa-quora"></i></a>{% endif %}
```

**Comment by monkeywithacupcake** [comment]
Can you make a PR @mkakodkar ? This seems like something that could be added.

**Comment by mkakodkar** [comment]
Oh yes I will do that

**Comment by mkakodkar** [comment]
https://github.com/alshedivat/al-folio/pull/40

**Comment by alshedivat** [comment]
Addressed in #40.
